IAM/Frequently_asked_questions.mediawiki

```diff
@@ -0,0 +1,49 @@
+=== Mozilla IAM FAQ (Frequently Asked Questions) ===
+
+==== '''Q''': ''What is Mozilla IAM?'' ====
+
+Mozilla IAM stands for Mozilla's Identity and Access Management. It's the internal login system that Mozilla uses to login to various web properties and systems.
+
+Usually, you'd use Mozilla IAM as a Mozilla Employee, or has a contributor with access to the tools and resources Mozilla uses day to day.
+An example of that would be Air Mozilla: https://air.mozilla.org
```

Comment:
Might we want to use Discourse as an example? AirMo still has the unsupported Lock. :-(

Reply:
done